pkg/data/management/ensure_admin_user.go

```diff
@@ -119,6 +119,15 @@ func createNewAdmin(client v3.Interface, length int) error {
 
 	fmt.Fprintf(os.Stdout, "New default admin user (%v):\n", admin.Name)
 	fmt.Fprintf(os.Stdout, "New password for default admin user (%v):\n%s\n", admin.Name, pass)
+
+	refreshAdmin, err := client.Users("").Get(admin.Name, v1.GetOptions{})
+	if err != nil {
+		return err
+	}
+	userKey, userValue := getAdminPrincipalLabel(admin)
+	newAdmin := refreshAdmin.DeepCopy()
```

Comment:
Why the deep copy? In case the update fails, existing user will be unaffected, right?

Reply:
Before updating a resource you do a deep copy to avoid changing the original resource (which I am told causes race conditions).

In this case, I was getting a "object has been changed try again" bug with out the deep copy and differently named var.